### PR TITLE
Add working Groups

### PR DESCRIPTION
**Pull request by klumb**

### CODE REVIEW

File: About/About/who-are-we.md [hunk 1]
```diff
@@ -1,7 +1,23 @@
...
+
+### Committees
 
 - The [Metrics Committee](https://chaoss.community/metrics) works on establishing implementation-agnostic metrics for measuring community activity, contributions, and health.
 
 - The [Software Committee](https://chaoss.community/software) works on producing integrated, open source software for analyzing software development, and definition of standards and models used in that software in specific use cases.
+
+###Workgroups
+
+The workgroups
```
Comment (GeorgLink) [inline About/About/who-are-we.md]: Line 13 might not be necessary?
> The workgroups

File: About/About/who-are-we.md [hunk 2]
```diff
@@ -1,7 +1,23 @@
...
+
+ The goal of the workgroups is to refine the metrics and to work with software implementations. The workgroups are built around the four categories of metrics that CHAOSS has identified. The workgroups are:
+
+- Growth Maturity and Decline
+
+- Diversity and Inclusion
```
Comment (GeorgLink) [inline About/About/who-are-we.md]: Line 17 and 19, the GMD and D&I workgroup names, could link out to the repositories that exist.

Also D&I is more mature workgroup and I would list it first.

File: Metrics/risk.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
-# Risk
+### Risk
 
-The Risk category informs how much risk an OSS community might pose. The evaluation of risk depends on situation and purpose. 
+he Risk category informs how much risk an OSS community might pose. The evaluation of risk depends on situation and purpose.
```
Comment (GeorgLink) [inline Metrics/risk.md]: Spelling error introduced at beginning of Line 3

Comment (GeorgLink) [reply Metrics/risk.md]: I went ahead and fixed it.

File: About/About/who-are-we.md (hunk 1 above)
Comment (klumb) [reply About/About/who-are-we.md]: Yes not necessary - typo

File: About/About/who-are-we.md (hunk 2 above)
Comment (klumb) [reply About/About/who-are-we.md]: Linking was the plan but I was planning on taking them to the website page for the work groups. page/pages didn't exist when I made the changes so I have left them without a link.

File: Metrics/risk.md (hunk 3 above)
Comment (klumb) [reply Metrics/risk.md]: thanks

File: About/About/who-are-we.md (hunk 2 above)
Comment (klumb) [reply About/About/who-are-we.md]: No issues moving D&I to the front

**Comment by klumb** [comment]
Requested changes made. I am holding off on linking work groups on the about page for the time being.

**Pull request merged**